pkg/operator/helpers/sa_syncer.go

```diff
@@ -78,30 +79,6 @@ func SATokenGetter(ctx context.Context, saName, saNamespace string, saClient kub
 	}
 }
 
-// SATokenCreater create the saToken of target sa.
-func SATokenCreater(ctx context.Context, saName, saNamespace string, saClient kubernetes.Interface) TokenGetterFunc {
```

Comment:
The `SATokenGetter` func contains logic in the `SATokenCreater` func, seems this is not necessary.

Reply:
/cc @zhiweiyin318 @xuezhaojun please take a look here